content/docs/setup/kubernetes/upgrading-istio/index.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Upgrading Istio
 description: Demonstrates how to upgrade the Istio control plane and data plane independently.
-weight: 70
+weight: 90
```

Comment:
upgrading should come right after the installation.  At the moment it appears like n information island.

Cheers
-steve

Reply:
Yeah, I intended for them to go after the series of "quick starts" as they are "installs" technically... but, you're right. I dropped it between the draft 'Advanced Install' and 'Installing the Sidecar'. Thanks!